cmd/embedded-cluster/restore.go

```diff
@@ -437,6 +459,39 @@ func pickBackupToRestore(backups []velerov1.Backup) *velerov1.Backup {
 	return latestBackup
 }
 
+// waitForRestoreCompleted waits for a Velero restore to complete.
+func waitForRestoreCompleted(ctx context.Context, restoreName string) (*velerov1.Restore, error) {
```

Comment:
I was trying to avoid this function name because it's not clear which "restore" we're waiting for. The EC restore? Velero restore? Any reason it couldn't remain as part of the `waitForDRComponent` function?

Reply:
The comment reads `waitForRestoreCompleted waits for a Velero restore to complete`. I could potentially just rename it to `waitForVeleroRestoreCompleted`? I didn't mind it as a a separate function to make `waitForDRComponent` a bit smaller and more readable.